engines/python/setup/djl_python/huggingface.py

```diff
@@ -125,10 +135,16 @@ def inference(self, inputs):
             input_map = decode(inputs, content_type)
             data = input_map.pop("inputs", input_map)
             parameters = input_map.pop("parameters", {})
+            outputs = Output()
 
+            if self.enable_streaming:
+                stream_generator = StreamingUtils.get_stream_generator("DeepSpeed")
```

Comment:
strange to use DeepSpeed generator in pure huggingface

Reply:
oops, sorry that's a typo. Will update. This worked because both DeepSpeed and HF share the same generator.